yarn.lock

```diff
@@ -5,10 +5,12 @@
 "@ava/babel-plugin-throws-helper@^2.0.0":
   version "2.0.0"
   resolved "https://registry.yarnpkg.com/@ava/babel-plugin-throws-helper/-/babel-plugin-throws-helper-2.0.0.tgz#2fc1fe3c211a71071a4eca7b8f7af5842cd1ae7c"
+  integrity sha1-L8H+PCEacQcaTsp7j3r1hCzRrnw=
```

Comment:
@Gerald1614 your `yarn` version is between `v1.10.0` and `v1.12.0` see `yarn`'s release [notes here](https://github.com/yarnpkg/yarn/releases). :laughing: try to avoid unrelated changes in your PR. You can e.g. `git checkout origin/master -- yarn.lock`.

If you want, you can create a new PR called "Add integrity-sha to yarn.lock". It doesn't seem to hurt.

Reply:
that was my adding with the integrity. i resolved it via the jnternal github resolver on this page. so there was no commit msg to ad